### PR TITLE
Qt/GameConfigWidget: Show revision specific INIs

### PR DESCRIPTION
**Pull request by spycrab**
Fixes https://bugs.dolphin-emu.org/issues/11676

### CODE REVIEW

**Comment by JosJuice** [comment]
Is there any particular reason to not use `ConfigLoaders::GetGameIniFilenames` (which would ensure consistency with the rest of Dolphin)?

**Comment by spycrab** [comment]
@JosJuice Now it does.

File: Source/Core/DolphinQt/Config/GameConfigWidget.cpp [hunk 1]
```diff
@@ -63,16 +60,18 @@ GameConfigWidget::GameConfigWidget(const UICommon::GameFile& game) : m_game(game
   LoadSettings();
   ConnectWidgets();
 
-  PopulateTab(m_default_tab, File::GetSysDirectory() + "GameSettings/", m_game_id, true);
-  PopulateTab(m_local_tab, File::GetUserPath(D_GAMESETTINGS_IDX), m_game_id, false);
+  PopulateTab(m_default_tab, File::GetSysDirectory() + "GameSettings/", m_game_id,
```
Comment (Techjar) [inline Source/Core/DolphinQt/Config/GameConfigWidget.cpp]: We have a `GAMESETTINGS_DIR` constant.

**Comment by Techjar** [comment]
Tested and working.


Not sure if we want to strip away the .ini extension to keep it consistent with the previous display, I'm indifferent on that.

**Pull request merged**